MIRest/src/com/cubeiosample/webservices/rest/jersey/MovieRentals.java

```diff
@@ -521,6 +523,32 @@ public int deleteRental() {
 		JSONObject rs = ros.executeUpdate(inventoryQuery, params);
 		return rs.getInt("num_updates");
 	}
+
+	public boolean validateUserAndPassword(String userName, String password) {
+		try {
+			String storesQuery = "select * from customer "
+					+ " where email = ? and password = ?";
+			JSONArray params = new JSONArray();
+			RestOverSql.addStringParam(params, userName);
+			RestOverSql.addStringParam(params, encodePassword(password));
+			JSONArray rs = ros.executeQuery(storesQuery, params);
+			if (rs == null || rs.length() < 1) {
+				return false;
+			}
+			return true;
+		} catch (Exception e) {
+			LOGGER.error("No user found for  user=" + userName + "; " + e.toString());
+		}
+		return false;
+	}
+
+	private String encodePassword(String password) {
+    	return Base64.getEncoder().encodeToString(password.getBytes());
```

Comment:
This should use a PasswordEncoder like in cubeui-backend. Otherwise we are storing passwords in a recoverable fashion. Even though this is a toy app, people may enter their actual passwords.